string[] task_map_names
std_msgs/Float64MultiArray[] task_map_goals
sensor_msgs/JointState start_state
sensor_msgs/JointState[] previous_solutions  # most recent previous solution should appear at the end